extensions/wikia/Rail/styles/RailElement.scss

```diff
@@ -0,0 +1,269 @@
+@import "skins/shared/color";
+
+$black: #000;
+$white: #FFF;
+$color-divider: #E6EBF2;
+$color-rail-background: mix($color-page, $black, 90%);
+@if $is-dark-wiki {
+    $color-rail-background: mix($color-page, $white, 90%);
+}
+
+
+@mixin wikia-rail-element() {
+    background-color: $color-rail-background;
+    padding: 20px;
+
+    zoom: 1;
+    margin-bottom: 10px;
+    position: relative;
+    word-wrap: break-word;
+}
+
+.wikia-rail {
+
+    float: right;
+    overflow: hidden;
+    padding: 0 10px 10px 0;
+    width: 300px;
+
+    h1 {
+        padding-left: 0;
+    }
+
+    .WikiaActivityModule {
+
+        @include wikia-rail-element;
+        font-size: 12px;
+        font-weight: bold;
+
+        a {
+            color: $color-links-default;
+        }
+
+        a:hover {
+            color: $color-links-default;
+            text-decoration: underline;
+            text-shadow: initial;
+        }
+
+        .activity-heading {
+            font-size: 24px;
+            font-weight: bold;
+            margin-bottom: 20px;
+        }
+
+        li em a {
```

Comment:
Maybe we can drop something here, simplify this selector?
And the other thing, you could place this rule earlier or later in comparison to the class/pseudo-class selectors, for better readability. But be aware that's not always applicable as attributes in CSS are overwriting themselves.